### PR TITLE
#5 and #17

### PR DESCRIPTION
**Pull request by dake3601**
### What does this PR do?

- Uploaded add-two-numbers.py and merge-two-binary-trees.py.

#### Related issues

#

#### Merge Checklist:

- [x] The branch is up-to-date (i.e. rebased) with the base branch
- [x] The link to the problem is the correct one
- [x] The description of the approach is clear
- [x] The code is executable
- [x] No code style issues according to [RoBorregos' C++ style guide](https://docs.google.com/document/d/1IwxK2oiHoX-eBNDntJe1bTFj6-GfTRXRtSbxhOBbvik/edit?usp=sharing)

### CODE REVIEW

**Comment by tomvik** [comment]
Please check all the tasks of the Merge Checklist that you've already completed.

File: Fall_2020/Week_1/add-two-numbers.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+b2.next = b3
+
+head = addTwoNumbers(a1, b1)
+
+string = ''
+while head != None:
+    string += str(head.val) + ' -> '
+    head = head.next
+
+print(string[:-4])
```
Comment (tomvik) [inline Fall_2020/Week_1/add-two-numbers.py]: Great work! Please do something similar as this for the other file.

File: Fall_2020/Week_2/merge-two-binary-trees.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            node1.left = node2.left
+            
+        if node1.right != None and node2.right != None and node1 != node2:
+            node1.right.val += node2.right.val
+        if node1.right == None and node2.right != None:
+            node1.right = node2.right
+        merge(node1.left, node2.left),merge(node1.right, node2.right)
+        return 
+    merge(t1, t2)
+    return t1
```
Comment (tomvik) [inline Fall_2020/Week_2/merge-two-binary-trees.py]: Please add a new line at the end of both files

**Review by tomvik — CHANGES_REQUESTED** [review]
Great work!

File: Fall_2020/Week_1/add-two-numbers.py (hunk 1 above)
Comment (dake3601) [reply Fall_2020/Week_1/add-two-numbers.py]: Done!

File: Fall_2020/Week_2/merge-two-binary-trees.py (hunk 2 above)
Comment (dake3601) [reply Fall_2020/Week_2/merge-two-binary-trees.py]: I added a new line at the end of both files but it is not visible.

**Review by tomvik — APPROVED** [review]
Excellent! Feel free to merge it. Thanks Keven!

**Pull request merged**